src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -95,17 +98,14 @@ const DeleteFreeAccountButton: FC = () => {
     history.push(`/orgs/${org.id}/org-settings/delete`)
   }
 
-  const handleShowWarning = () => {
+  const showRemoveUsersWarning = () => {
```

Comment:
❤️ 

much better method names. It's the little things

Reply:
🥳